ui-tests/src/test/java/org/openmrs/reference/AddFormTest.java

```diff
@@ -18,22 +16,40 @@
  */
 public class AddFormTest extends TestBase {
     private HomePage homePage;
+    private static String name = "newFormTest";
+    private static String description = "description of new form";
+    private static  String version = "1.2";
+
+    private static final By NEW_HTML_FORM = By.cssSelector("a[href='htmlForm.form']");
```

Comment:
do you need this field here? and if yes, can u make use of https://github.com/openmrs/openmrs-distro-referenceapplication/pull/360/files#diff-268be05503616ccdc23bf47f929298bcf3eb451ca81e8b2e1d0b9dd90ca0315dR8

Reply:
This line of code is looks similar with the above code  , unless am getting you wrongly